earthpy/spatial.py

```diff
@@ -80,8 +81,8 @@ def normalized_diff(b1, b2):
 # TODO: include a no data value here if provided
 
 def stack_raster_tifs(band_paths, out_path, arr_out=True):
-    """Take a list of raster paths and turn into an output raster stack in numpy format.
-    Note that this function depends upon the stack() function.
+    """Take a list of raster paths and turn into an output raster stack in
+    numpy format. Note that this function depends upon the stack() function.
```

Comment:
Would it be more precise to say "as a numpy array" instead of "in numpy format"?

Reply:
yes ! much more precise !

Reply:
fixed!!